src/main/java/jupyter/MIMETypes.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 Netflix
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jupyter;
+
+public class MIMETypes {
+  public static final String TEXT = "text/plain";
+  public static final String HTML = "text/html";
+}
```

Comment:
Side note -- custom mimetypes have been starting to be placed [in the jupyter docs](http://jupyter.readthedocs.io/en/latest/reference/mimetype.html#listing-of-custom-mimetypes-used-for-display).